packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalkContainer.js

```diff
@@ -174,21 +148,21 @@ class QuickTalkContainer extends React.Component {
 
       const authorization = await getBearerToken(authClient)  // Check bearer token to ensure session hasn't timed out
```

Comment:
I think I’ve pointed this bug out before but this will return an empty string if you call it before you call `checkCurrent`. See #2986.